spring-session-data-redis/src/main/java/org/springframework/session/data/redis/ReactiveRedisSessionRepository.java

```diff
@@ -36,7 +36,7 @@
  * A {@link ReactiveSessionRepository} that is implemented using Spring Data's
  * {@link ReactiveRedisOperations}.
  *
- * @author Vedran Pavic
+ * @author Kai Zhao
```

Comment:
Please do not remove existing authors and instead add you name in addition.
```suggestion
 * @author Vedran Pavic
 * @author Kai Zhao
```

Reply:
Sorry, I submit my contribution for the first time. I will modify it according to the suggestions. Thank you